test/simplify_qdq_test.cpp

```diff
@@ -0,0 +1,636 @@
+        auto d5  = add_quantize_op(mm, "dequantizelinear", q1, l2, l1);
+        auto c1  = mm.add_instruction(migraphx::make_op("convolution",
+                                                       {{"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"dilation", {1, 1}},
+                                                        {"group", 1},
+                                                        {"padding_mode", 0}}),
+                                     d5,
+                                     d1);
+        auto bc1 = mm.add_instruction(
+            migraphx::make_op("broadcast", {{"axis", 1}, {"dims", {1, 1280, 7, 7}}}), d2);
+        auto a1 = mm.add_instruction(migraphx::make_op("add"), c1, bc1);
+        auto q2 = add_quantize_op(mm, "quantizelinear", a1, l2, l1);
+        auto d6 = add_quantize_op(mm, "dequantizelinear", q2, l2, l1);
+        auto ap = mm.add_instruction(migraphx::make_op("pooling",
+                                                       {{"mode", "average"},
+                                                        {"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"lengths", {7, 7}},
+                                                        {"ceil_mode", 0}}),
+                                     d6);
+        auto q3 = add_quantize_op(mm, "quantizelinear", ap, l2, l1);
+        auto d7 = add_quantize_op(mm, "dequantizelinear", q3, l2, l1);
+        auto rs = mm.add_instruction(migraphx::make_op("reshape", {{"dims", {1, -1}}}), d7);
+        auto q4 = add_quantize_op(mm, "quantizelinear", rs, l2, l1);
+        auto d8 = add_quantize_op(mm, "dequantizelinear", q4, l2, l1);
+        auto dot =
+            mm.add_instruction(migraphx::make_op("dot", {{"alpha", 1}, {"beta", 0}}), d8, d4);
+        auto q5  = add_quantize_op(mm, "quantizelinear", dot, l2, l1);
+        auto d9  = add_quantize_op(mm, "dequantizelinear", q5, l2, l1);
+        auto mb1 = mm.add_instruction(
+            migraphx::make_op("multibroadcast", {{"output_lens", {1, 1000}}}), d3);
+        auto a2 = mm.add_instruction(migraphx::make_op("add"), d9, mb1);
+        mm.add_return({a2});
+
+        return mm;
+    };
+
+    auto mod1 = create_module();
+    auto mod2 = create_module();
+
+    run_pass(mod2);
+
+    auto match_qdq = migraphx::match::name("dequantizelinear")(
+        migraphx::match::arg(0)(migraphx::match::name("quantizelinear")));
+    auto ins1 = migraphx::match::find_match(mod1, match_qdq);
+    auto ins2 = migraphx::match::find_match(mod2, match_qdq);
+
+    EXPECT((ins1.result != mod1.end()) and (ins2.result == mod2.end()));
+    EXPECT(any_of(mod1, &is_convolution));
+    EXPECT(none_of(mod2, &is_convolution));
+    EXPECT(any_of(mod1, &is_dot));
+    EXPECT(none_of(mod2, &is_dot));
```

Comment:
Please add the unit test as I mentioned above. Also create a similar unit test for the convolution.